app/controllers/organizations/services_controller.rb

```diff
@@ -31,10 +31,12 @@ def filter_params
     end
 
     # :reek:UtilityFunction
+    # :nocov:
```

Comment:
Since filtering is not fully implemented yet and I do not want to bloat this PR even more, I will need to implement it in a later PR